org.scala-ide.sdt.core.tests/src/org/scalaide/core/sbtbuilder/CompilerBridgeStoreTest.scala

```diff
@@ -0,0 +1,28 @@
+package org.scalaide.core.sbtbuilder
+
+import org.junit.Test
+import org.scalaide.core.internal.project.ScalaInstallation.platformInstallation
+import org.scalaide.core.internal.ScalaPlugin
+import org.junit.Assert
+
+class CompilerBridgeStoreTest {
+
+  @Test
+  def platformCompilerBridgeWorks(): Unit = {
+    val store = ScalaPlugin().compilerBridgeStore
+    store.purgeCache()
+
+    Assert.assertTrue("successful compiler bridge compilation", store.compilerBridgeFor(platformInstallation)(null).isRight)
+    Assert.assertEquals("Zero hits and one miss", (0, 1), store.getStats)
```

Comment:
The first number is hits, the second one is misses. It is explained in Scaladoc of `getStats`. Too lazy to convert that to convert that to a case class.